app/helpers/teacher_sets_helper.rb

```diff
@@ -1,11 +1,12 @@
 # frozen_string_literal: true
 
 module TeacherSetsHelper
+
   # Make request input body to create teacherset document in elastic search.
   # Input param ts_obj eg: <TeacherSet:0x00007fd79383a640 id: 350, title: "Step",call_number: "Teacher",
   # description: "Book", details_url: "http://catalog.nypl.org/record=b21378444~S1","updated-7571-Step up to the plate, Maria Singh">
   # Expected response from this method {:title=>"Step Up", :description=> "Book", :contents=>"Step Up", :id=>350}
-
+  
   def teacher_set_info(ts_obj)
```

Comment:
Maybe should be an instance method?

Reply:
My mistake! Probably don't need to pass in `ts_obj` then and can just use `self` in the method?